package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@klarna/ui-css-components",
-  "version": "5.6.4",
+  "version": "5.7.0",
```

Comment:
Usually you bump the version after the merge since can't know what will be the current version when you merge